crud.go

```diff
@@ -393,6 +401,9 @@ func (bucket *WalrusBucket) waitAfterWrite(seq uint64, opt sgbucket.WriteOptions
 func (bucket *WalrusBucket) write(k string, exp uint32, raw []byte, opt sgbucket.WriteOptions) (seq uint64, err error) {
 	bucket.lock.Lock()
 	defer bucket.lock.Unlock()
+	if len(raw) > 20000000{
```

Comment:
Magic number nested pretty deep in the code. Could we pull this out into a constant?